app-backend/app/src/main/scala/project/Routes.scala

```diff
@@ -101,8 +101,16 @@ trait ProjectRoutes extends Authentication
 
   def listProjectScenes(projectId: UUID): Route = authenticate { user =>
     (withPagination & sceneQueryParameters) { (page, sceneParams) =>
-      complete {
-        Projects.listProjectScenes(projectId, page, sceneParams, user)
+      if (page.offset > -1) {
```

Comment:
I think this needs to happen at a higher level -- possibly by its own directive or some sort that can be mixed in the `withPagination` directive, otherwise there will be quite a bit of duplicate code. This will need to be a check on every paginated endpoint, not just these two.